src/pyproject.toml.m4

```diff
@@ -6,7 +6,6 @@ requires = [
     # Note that PEP517/518 have no notion of optional sage_spkg dependencies:
     # https://github.com/pypa/pip/issues/6144
      esyscmd(`sage-get-system-packages install-requires-toml \
-        sage_conf      \
         setuptools \
```

Comment:
Line 5 says "Some version of sage-conf is required". But `sage_conf` is removed from line 9. 

Then would `pip install sagemath-standard`  work?

Reply:
yes, it's still working as can be seen from the conda ci workflow.

Thanks for catching the outdated comment. I've removed it now.